app/assets/stylesheets/component_guide/application.scss

```diff
@@ -2,12 +2,10 @@
 
 @import "govuk_publishing_components/components/helpers/govuk-frontend-settings";
 @import "govuk_publishing_components/components/helpers/markdown-typography";
-@import "govuk-frontend/settings/all";
-@import "govuk-frontend/tools/all";
-@import "govuk-frontend/helpers/all";
-@import "govuk-frontend/core/all";
-
-@include _govuk-font-face-nta;
```

Comment:
does this no longer need importing?

Reply:
Nope, that was a hack to use the new font in the component guide app as it wasn't loaded because of the compatibility flags. Now that we have them turned off the font comes by default and this is no longer needed.